benchmarks/src/main/java/zipkin/benchmarks/MetricsBenchmarks.java

```diff
@@ -45,9 +47,9 @@
   static final int LONG_SPAN = 5000;
   static final int MEDIUM_SPAN = 1000;
   static final int SHORT_SPAN = 500;
-
+  @Autowired private MeterRegistry registry;
```

Comment:
I don't think you can autowire here because this is a JMH test that knows nothing about Spring.

Reply:
My bad. I should have looked closer. Can we include Micrometer and do the benchmarks with a SimpleRegistry or remove this one?